src/Accessor/PropertyAccessor.php

```diff
@@ -28,7 +29,11 @@ public function __construct(string $property)
      */
     public function getValue($object)
     {
-        $class = get_class($object);
+        if (true === class_exists('Doctrine\Common\Persistence\Proxy') && true === $object instanceof Proxy) {
```

Comment:
@TiMESPLiNTER please do not use true === if the return value of a method is a boolean